src/containers/Bookings.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import Search from '../components/Search.js';
-// import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
```

Comment:
Make sure to bring this line of code back in as I mentioned in the the `Results.js` file.